developer-extension/src/background/utils.ts

```diff
@@ -19,3 +19,10 @@ function evaluateCodeInline(tabId: number, code: (arg?: string) => void, arg?: s
     }`,
   })
 }
+
+export function generateUUID(placeholder?: string): string {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I know this is a copy-paste from somewhere else, but this function could be a bit more readable

Reply:
I removed it and use our existing function in core/src/tools/utils